This patch file needs to be placed inside jhalfs/custom/. The patch
is automatically applied by the './jhalfs run', and what it does is
to modify some of the instructions of the LFS book so that packages
are installed nicely on the GoboLinux tree.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

Index: chapter06/ncurses.xml
===================================================================
--- chapter06/ncurses.xml	(revision 10416)
+++ chapter06/ncurses.xml	(working copy)
@@ -116,6 +116,8 @@
     non-wide-character Ncurses libraries. Trick such applications into linking with
     wide-character libraries by means of symlinks and linker scripts:</para>
 
+<screen><userinput remap="install">mkdir -p /usr/lib/pkgconfig</userinput></screen>
+
 <screen><userinput remap="install">for lib in ncurses form panel menu ; do
     rm -vf                    /usr/lib/lib${lib}.so
     echo "INPUT(-l${lib}w)" &gt; /usr/lib/lib${lib}.so
Index: chapter06/adjusting.xml
===================================================================
--- chapter06/adjusting.xml	(revision 10416)
+++ chapter06/adjusting.xml	(working copy)
@@ -57,7 +57,7 @@
 
   <para os="d">Now make sure that we're setup to use the correct startfiles:</para>
 
-<screen os="e"><userinput>grep -o '/usr/lib.*/crt[1in].*succeeded' dummy.log</userinput></screen>
+<screen os="e"><userinput>grep -o '/lib.*/crt[1in].*succeeded' dummy.log</userinput></screen>
 
   <para os="f">If everything is working correctly, there should be no errors,
     and the output of the last command will be:</para>
Index: chapter07/network.xml
===================================================================
--- chapter07/network.xml	(revision 10416)
+++ chapter07/network.xml	(working copy)
@@ -130,7 +130,7 @@
 
     <para>The following command creates a sample file for the
     <emphasis>eth0</emphasis> device with a static IP address:</para>
-
+<!--
 <screen><userinput>cd /etc/sysconfig/
 cat &gt; ifconfig.eth0 &lt;&lt; "EOF"
 <literal>ONBOOT=yes
@@ -141,7 +141,7 @@
 PREFIX=24
 BROADCAST=192.168.1.255</literal>
 EOF</userinput></screen>
-
+-->
     <para>The values of these variables must be changed in every file to match
     the proper setup.</para>
 
Index: chapter06/coreutils.xml
===================================================================
--- chapter06/coreutils.xml	(revision 10436)
+++ chapter06/coreutils.xml	(working copy)
@@ -49,7 +49,7 @@
     boundaries correctly even in multibyte locales. The following patch
     fixes this non-compliance and other internationalization-related bugs:</para>
 
-<screen><userinput remap="pre">patch -Np1 -i ../&coreutils-i18n-patch;</userinput></screen>
+<screen><userinput remap="pre">echo patch -Np1 -i ../&coreutils-i18n-patch;</userinput></screen>
 
     <note>
       <para>In the past, many bugs were found in this patch. When reporting new
Index: chapter06/gcc.xml
===================================================================
--- chapter06/gcc.xml.orig	2017-04-03 08:51:09.618819202 -0300
+++ chapter06/gcc.xml	2017-04-03 08:51:33.937819658 -0300
@@ -191,7 +191,7 @@ ln -sfv ../../libexec/gcc/$(gcc -dumpmac
     href="adjusting.xml"
     xpointer="xpointer(//*[@os='g'])"/>
 
-<screen><userinput>grep -B4 '^ /usr/include' dummy.log</userinput></screen>
+<screen><userinput>grep -B4 '^ /usr/include\|^ /usr/local/include' dummy.log</userinput></screen>
 
     <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
     href="adjusting.xml"
